app/src/main/java/cmput301f17t13/com/catisadog/activities/LoginActivity.java

```diff
@@ -0,0 +1,355 @@
+package cmput301f17t13.com.catisadog.activities;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.annotation.TargetApi;
+import android.content.pm.PackageManager;
+import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.app.LoaderManager.LoaderCallbacks;
+
```

Comment:
Why is there blank space between these imports (and others)?

Reply:
Not sure why here (didn't make this file), but Android studio generally splits them by package highest order of package.

Reply:
Okay. Let's stick with Android Studio's convention.